### ISSUE TITLE
Needs ExcludeFromHistory property, for fields.

### ISSUE THREAD

**Issue by BrianMinister** [Arch/AutoHistory#30]
When recording history, there are fields that I don't want their real data recorded in history. These are like Personal Identifying data, or passwords (even after being hashed).

Though making this an extension method that would work in the OnModelCreating call, it would also be nice to see it as an attribute for the model fields themselves.

``` [ExcludeFromHistory(ReplacementValue="")] ```

**Comment by xyting** [comment]
Good idea, this is a real requirement. You can PR for this or I will done this in next release.

**Comment by xyting** [comment]
When release new version, this should be supported.

**Comment by BrianMinister** [comment]
I started on this, then realized that Newtonsoft's library is being used.
So I am testing using their `[JsonIgnore]` attribute for the field level access.

I suspect that a whole model implementing this would cover the model, if the serialization code isn't ignoring these attributes.

**Comment by coolhome** [comment]
I had a case where i needed to exclude binary data, however a more advanced feature could be using a hash for the binary data instead of the difference.

I completely agree with an attribute for `ExcludeFromHistory`.

This would be super easy to implement, anyone have a problem if I implement it?
